README.md

```diff
@@ -7,13 +8,24 @@ Cineast is written in Java and uses [CottontailDB](https://github.com/https://gi
 ## Building Cineast
 Cineast can be built using [Gradle](http://gradle.org/). Building and running it is as easy as
 ```
- git clone --recursive https://github.com/vitrivr/cineast.git cineast
- cd cineast
- ./gradlew :cineast-core:generateProto
- ./gradlew cineast-runtime:fatJar
- java -jar cineast-runtime/build/libs/cineast-runtime-x.x-full.jar cineast.json
+$> git clone --recursive https://github.com/vitrivr/cineast.git cineast
+$> cd cineast
+$> ./gradlew :cineast-core:generateProto
+$> ./gradlew cineast-runtime:fatJar
+$> java -jar cineast-runtime/build/libs/cineast-runtime-x.x-full.jar cineast.json
  ```
 
+## Generate OpenApi Specification
+
+If you need to rebuild the OpenApi Specification (OAS), there is a gradle task for this purpose:
+
+```
+$> ./gradlew -PcineastConfig=<path/to/your/config> generateOpenApiSpecs
+```
+
+You can omit `-PcineastConfig`, then the default config (`cineast.json`) is used.
+As a result, the OAS is stored at `docs/swagger.json`
```

Comment:
I would rather use `openapi.json` or `oas.json` to highlight the support for OAS +3. The name `swagger.json` is -by convenvtion- used for OAS 2.0